### DISCUSSION TITLE
Sort test execution order

### DISCUSSION THREAD

**Discussion by StephaneColson** [currents-dev/cypress-currents-reporter#82] [Ideas]
Hello currents team,

In a test run detail (page `https://app.currents.dev/instance/xxx`), when no failures then the steps are displayed in the run order. When some steps fails, the ones with errors are displayed first, then the other steps in execution order.

It would be nice to be able to sort this list of steps in order to display in execution order even with failures.

Thx

**Comment by agoldis** [comment]
Hey @StephaneColson added to our roadmap - will let you know when it's ready. Thanks for suggesting!
